releasenotes/notes/otlp-peer-service-15c0b4a105.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Support setting _dd.peer.service.source in OTLP ingest endpoint.
```

Comment:
`_dd.peer.service.source` is an internal tag which doesn't have any meaning to the end user. If you were to rephrase this for a user, how would you write it? How does this change affect the user experience? Remember, this change log is not for internal developers working at Datadog, it's for agent users.

Reply:
Great point, for now I updated it to match https://github.com/DataDog/datadog-agent/pull/16103/files#diff-bfd776664558b32d9883a48d1aaaae7d54eeb278e8c51182ec7680d8d69ac2aa. I think it would be better to link to a public reference of APM peer service if we have one /cc @jdgumz